app/server.js

```diff
@@ -1,10 +1,16 @@
 const app = require('express')();
 const nconf = require('nconf');
 const globSync = require('glob').sync;
+const morgan = require('morgan');
 
 const { errorHandler, notFound } = require('./middleware/error_handler');
 
 const initApp = () => {
+  const env = process.env.NODE_ENV || 'dev';
+  if (env === 'dev') {
+    app.use(morgan(env));
+  }
```

Comment:
it'll be great if we implement the following feature:
https://github.com/expressjs/morgan#log-file-rotation

Reply:
I create a ticket for it

Reply:
I've created an issue